File: app/src/main/java/com/revotechs/calculator/tools/DBHelper.java
package com.revotechs.calculator.tools;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;
import android.util.Log;

/**
 * Project Calculator
 * <p>
 * Created on 30.05.2017
 *
 * @author CriticalGnome
 */

public class DBHelper extends SQLiteOpenHelper {
    public DBHelper(Context context) {
        super(context, "CalcDB", null, 1);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        Log.d("Calculator", "--- onCreate database ---");
        db.execSQL("create table history (id integer primary key autoincrement, date text, expression text, result text, comment text, locked integer);");
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/revotechs/calculator/tool/DBHelper.kt
package com.revotechs.calculator.tool

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import android.util.Log

/**
 * Project Calculator
 * Created on 30.05.2017
 * @author CriticalGnome
 */
class DBHelper(context: Context) : SQLiteOpenHelper(context, "CalcDB", null, 1) {

    override fun onCreate(db: SQLiteDatabase) {
        Log.d("Calculator", "--- onCreate database ---")
        db.execSQL("create table history (id integer primary key autoincrement, date text, expression text, result text, comment text, locked integer);")
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {

    }
}
